ironfish/src/wallet/wallet.ts

```diff
@@ -592,10 +598,11 @@ export class Wallet {
     }
 
     if (this.chainProcessor.hash === null) {
-      const latestHeadHash = await this.getLatestHeadHash()
-      Assert.isNotNull(latestHeadHash, `scanTransactions: No latest head hash found`)
+      const latestHead = await this.getLatestHead()
+      Assert.isNotNull(latestHead, `scanTransactions: No latest head hash found`)
```

Comment:
Is it worth it to change this assert error to be more precise (` no head found`, because the code finds the head and then accesses the hash) or nah?

Reply:
Good catch, I think it's worth it!